### PR TITLE
PHPStan 1.7.13

### PR DESCRIPTION
**Pull request by derrabus**

### CODE REVIEW

File: phpstan-persistence2.neon [hunk 1]
```diff
@@ -16,6 +16,7 @@ parameters:
             path: lib/Doctrine/ORM/Tools/Console/ConsoleRunner.php
 
         - '/^Class Doctrine\\DBAL\\Platforms\\(PostgreSQL|SQLServer|SQLAnywhere)Platform not found\.$/'
+        - '/^Call to method \w+\(\) on an unknown class Doctrine\\DBAL\\Platforms\\(PostgreSQL|SQLServer|SQLAnywhere)Platform\.$/'
```
Comment (SenseException) [inline phpstan-persistence2.neon]: Why is this happening? Doesn't PHPStan find the DBAL classes?

Comment (derrabus) [reply phpstan-persistence2.neon]: Those classes don't exist in DBAL 3.

Comment (derrabus) [reply phpstan-persistence2.neon]: Well, the SQLAnywhere one at least. The others have been added back in the meantime. I'll try to simplify the rule.

Comment (derrabus) [reply phpstan-persistence2.neon]: Done.

**Review by SenseException — APPROVED** [review]

**Pull request merged**